onnx/defs/tensor/defs.cc

```diff
@@ -778,6 +803,36 @@ Example 2:
   ]
 )DOC";
 
+inline int64_t clamp(int64_t val, int64_t low, int64_t high) {
```

Comment:
this can be converted in 1 line:
(val < low) ? low : (val > high) ? high : val;

also I suggest change low/high to min/max for better readability

another suggestion is you can simply use this 1 line instead of creating a inline function